web/tests/firestore/firestore.test.js

```diff
@@ -0,0 +1,22 @@
+import { auth } from '../../firebase';
+
+let user;
+
+beforeAll(() =>
+  auth.createUserWithEmailAndPassword('user@example.com', '12345678').then((userCredential) => {
+    console.log('user successfully created!');
+    ({ user } = userCredential);
+  }).catch((error) => {
+    console.err('failed to create user: ', error);
+  }));
+
+test('basic test', () => {
+  expect(user.email).toBe('user@example.com');
+});
+
+afterAll(() =>
+  user.delete().then(() => {
```

Comment:
`user.delete()` This shouldn't be done on global cleanups - `afterAll`s should be used for resource remove (shutting down connections, etc), everything else should be contained either within each individual test or in `afterEach` clauses to avoid tests depending on each other. Right now it might be fine, but as the test suite grows this will be problematic